server/historian/packages/historian/README.md

```diff
@@ -0,0 +1,37 @@
+# historian
+
```

Comment:
Follow the routerlicious and routerlicious-base package for readme's. And move this one to top level folder.